cmd/drone-docker/main.go

```diff
@@ -171,6 +171,11 @@ func main() {
 			Usage:  "docker repository",
 			EnvVar: "PLUGIN_REPO",
 		},
+		cli.StringSliceFlag{
+			Name:   "Labels",
+			Usage:  "labels",
+			EnvVar: "PLUGIN_LABELS",
```

Comment:
`labels` in the yaml is a reserved keyword, so we'll need to choose another attribute name

Reply:
I figure only `!label` is reserved?

Reply:
container labels are reserved per the docker-compose specification. See https://docs.docker.com/compose/compose-file/#labels-1. Drone therefore uses these fields for special purposes

Reply:
note that `dns` is also a reserved attribute, so you will notice it uses `custom_dns` with the `custom_` prefix. I would therefore recommend we follow the same pattern with `custom_labels`

Reply:
Is this what you had in mind ;)?